src/routes/Landing.js

```diff
@@ -114,7 +111,9 @@ class Landing extends Component {
             <Fragment>
                 { iframeFlag
                     ? <Fragment>
-                        <iframe sandbox='allow-scripts allow-same-origin' srcDoc='<script type="text/javascript"> window.insights.chrome.auth.login(); </script>' style={ this.iframeStyle } onLoad={ this.handleIframeToggle }/>
+                        <iframe sandbox='allow-scripts allow-same-origin'
```

Comment:
Can you make each attribute its own line?
```
<iframe
    sandbox='allow-scripts allow-same-origin'
    srcDoc='<script type="text/javascript">window.insights.chrome.auth.login();</script>'
    style={ this.iframeStyle }
    onLoad={ this.handleIframeToggle }
/>
```
or
```
<iframe
    sandbox='allow-scripts allow-same-origin'
    srcDoc='<script type="text/javascript">window.insights.chrome.auth.login();</script>'
    style={ this.iframeStyle }
    onLoad={ this.handleIframeToggle }/>
```

Reply:
done